### PR TITLE
Libibumad bump to 1.3.10.2

### PR DESCRIPTION
**Pull request by qiankehan**
Version bump to 1.3.10.2 to resolve **infiniband/umad_types.h** missing issue and remove old version libibumad-1.3.9.

### CODE REVIEW

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="This is a sample skeleton ebuild file"
```
Comment (arthurzam) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild]: Please fix the `DESCRIPTION=`. The original was good:
`DESCRIPTION="OpenIB User MAD lib functions which sit on top of the user MAD kernel modules"`

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+HOMEPAGE="https://www.openfabrics.org"
+SRC_URI="https://www.openfabrics.org/downloads/management/${P}.tar.gz"
+
+SLOT="0"
+
+KEYWORDS="amd64"
+IUSE="+pic static"
+
+#DEPEND="${RDEPEND}"
```
Comment (arthurzam) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild]: Are you sure `sys-fabric/libibverbs` isn't a dependency now?
If not, you can remove this comment.

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	local myeconfargs=(
+		$(use_enable static)
+		$(use_with pic)
+	)
+
+	econf "${myeconfargs[@]}"
+}
+
+src_compile() {
+	emake
```
Comment (arthurzam) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild]: This is the default `src_compile`. You can remove this whole function.

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```
Comment (arthurzam) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild]: This is the default `src_install`. You can remove this whole function.

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+EAPI=7
+
+DESCRIPTION="This is a sample skeleton ebuild file"
+HOMEPAGE="https://www.openfabrics.org"
+SRC_URI="https://www.openfabrics.org/downloads/management/${P}.tar.gz"
+
+SLOT="0"
+
+KEYWORDS="amd64"
```
Comment (arthurzam) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild]: In new versions it must be unstable, meaning `~amd64`.
Is only `amd64` supported now? I think we can put here the unstable version of the old ebuild, meaning:
`KEYWORDS="~amd64 ~x86 ~amd64-linux"`

File: sys-fabric/libibumad/metadata.xml [hunk 6]
```diff
@@ -10,4 +10,12 @@ OpenIB User MAD library functions which sit on top of the user MAD modules in
 the kernel.  These are used by the IB diagnostic and management tools, including
 OpenSM.
 </longdescription>
+<use>
+	<flag name="pic">
+		Force shared libraries to be built as PIC (this is slower)
```
Comment (arthurzam) [inline sys-fabric/libibumad/metadata.xml]: Make this as a one line, looking like:
`<flag name="pic">Force shared libraries to be built as PIC (this is slower)</flag>`

File: sys-fabric/libibumad/metadata.xml [hunk 7]
```diff
@@ -10,4 +10,12 @@ OpenIB User MAD library functions which sit on top of the user MAD modules in
...
+	</flag>
+	<flag name="static">
+		Provides a static build
```
Comment (arthurzam) [inline sys-fabric/libibumad/metadata.xml]: Make this as a one line, looking like:
`<flag name="static">Provides a static build</flag>`

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild (hunk 2 above)
Comment (qiankehan) [reply sys-fabric/libibumad/libibumad-1.3.10.2.ebuild]: Yes. No probem when built without libibverbs

**Comment by qiankehan** [comment]
@arthurzam LICENSE has been updated and drop the old version removal commit. Please help review it :)

**Comment by qiankehan** [comment]
Any update ?

**Comment by qiankehan** [comment]
ping

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="https://www.openfabrics.org"
+LICENSE="|| ( BSD GPL-2 )"
+SRC_URI="https://www.openfabrics.org/downloads/management/${P}.tar.gz"
+
+SLOT="0"
+
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="+pic static"
+
+#DEPEND="${RDEPEND}"
```
Comment (ajakk) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild:16]: Let's not add commented code in ebuilds unless it serves some purpose, I don't think commenting this here adds any value.

File: sys-fabric/libibumad/libibumad-1.3.10.2.ebuild [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+DESCRIPTION="OpenIB User MAD lib functions which sit on top of the user MAD kernel modules"
+HOMEPAGE="https://www.openfabrics.org"
+LICENSE="|| ( BSD GPL-2 )"
+SRC_URI="https://www.openfabrics.org/downloads/management/${P}.tar.gz"
+
+SLOT="0"
+
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="+pic static"
```
Comment (ajakk) [inline sys-fabric/libibumad/libibumad-1.3.10.2.ebuild:14]: Why did you add these USEs?

File: sys-fabric/libibumad/metadata.xml [hunk 10]
```diff
@@ -10,4 +10,10 @@ OpenIB User MAD library functions which sit on top of the user MAD modules in
 the kernel.  These are used by the IB diagnostic and management tools, including
 OpenSM.
 </longdescription>
+<use>
+	<flag name="pic">Force shared libraries to be built as PIC (this is slower)
```
Comment (ajakk) [inline sys-fabric/libibumad/metadata.xml:14]: These <flag></flag> blocks can be one line.

**Review by ajakk — COMMENTED** [review]
Still there?

**Comment by ajakk** [comment]
Seems no one is there
